docs/eventing/sources/apiserversource/getting-started/README.md

```diff
@@ -81,86 +85,86 @@ Before you can create an API server source, you must install Knative Eventing an
-     resources:
-       - apiVersion: v1
-         kind: Event
-     sink:
-       ref:
-         apiVersion: v1
-         kind: Service
-         name: <sink>
-    EOF
-    ```
-    where;
-    - `<apiserversource>` is the name of the source that you want to create.
+    - `<cluster-role-binding>` is the name of the ClusterRoleBinding that you want to create.
+    - `<cluster-role>` is the name of the ClusterRole that you created in step 3 above.
+    - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
     - `<namespace>` is the name of the namespace that you created in step 1 above.
-    - `<service-account>` is the name of the service account that you created in step 2 above.
-    - `<sink>` is the name of the Knative service that you want to use as a sink. A service is used here as an example, however you can use any supported PodSpecable object by updating the `kind` from `Service` to another object type.
-
-
-
-
-
 
-6. Create events by launching a test pod in your namespace:
+1. Create the ApiServerSource object by running the command:
+
+    === "kn"
+
+        ```bash
+        kn source apiserver create <apiserversource> \
+          --namespace <namespace> \
+          --mode "Resource" \
+          --resource "Event:v1" \
+          --service-account <service-account> \
+          --sink <sink>
+        ```
+        Where:
+
+        - `<apiserversource>` is the name of the source that you want to create.
+        - `<namespace>` is the name of the namespace that you created in step 1 above.
+        - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
+        - `<sink>` is the name of the PodSpecable object that you want to use as a sink.
+
+    === "YAML"
+
+        ```yaml
+        kubectl create -f - <<EOF
+        apiVersion: sources.knative.dev/v1
+        kind: ApiServerSource
+        metadata:
+         name: <apiserversource>
+         namespace: <namespace>
+        spec:
+         serviceAccountName: <service-account>
+         mode: Resource <!-- should this be a placeholder? -->
+         resources:
+           - apiVersion: v1
+             kind: Event
+         sink:
+           ref:
+             apiVersion: v1
+             kind: Service
+             name: <sink>
+        EOF
+        ```
+        Where:
+
+        - `<apiserversource>` is the name of the source that you want to create.
+        - `<namespace>` is the name of the namespace that you created in step 1 above.
+        - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
+        - `<sink>` is the name of the Knative Service that you want to use as a sink. A Service is used here as an example, however you can use any supported PodSpecable object by updating the `kind` from `Service` to another object type.
+
+
+## Verify the ApiServerSource object
+
+1. Create events by launching a test pod in your namespace by running the command:
```

Comment:
I would say "Make Kubernetes API server create events by ...".

Because, Kube Api server creates these events and the APIServerSource collects them and sends them to the sink.